www/source/partials/docs/_commands-cli.html.md.erb

```diff
@@ -721,10 +721,255 @@ Exports the package to the specified format
 
 **ARGS**
 
-    <FORMAT>       The export format (ex: aci, cf, docker, kubernetes, mesos, or tar)
+    <FORMAT>       The export format (ex: aci, cf, docker, helm, kubernetes, mesos, or tar)
     <PKG_IDENT>    A package identifier (ex: core/redis, core/busybox-static/1.42.2) or filepath
                     to a Habitat Artifact (ex: /home/acme-redis-3.0.7-21120102031201-x86_64-linux.hart)
 
+**Read More**
+
+- [docker](#hab-pkg-export-docker) Creates a Docker image from a set of Habitat packages
+- [kubernetes](#hab-pkg-export-k8s) Creates a Docker image and Kubernetes manifest for a Habitat package
```

Comment:
Either pick `for a Habitat package` or `for the specified Habitat package` as used below?

Reply:
It's a copy&paste from the actual `--help` output and looking at others, it seems it's supposed to be exactly the same. right?

Reply:
Yes, I understand that part, just side by side they seem awkward. Was just a nitpick suggestion. :)